third_party/jemalloc.BUILD

```diff
@@ -29,7 +29,7 @@ JEMALLOC_BUILD_COMMAND = """
   export OBJCOPY=$$(absolutize $(OBJCOPY))
   export CFLAGS=$(CC_FLAGS)
   export CXXFLAGS=$(CC_FLAGS)
-  export LTOFLAGS="$$([ "$$(uname)" = "Linux" ] && echo "-flto=thin")" # todo: on next clang toolchain upgrade, check if it's fixed and we can re-enable thinlto on mac
+  export LTOFLAGS="$$([ "$$(uname)" = "Linux" ] && echo "-flto=thin -Wl,--thinlto-jobs=all")" # todo: on next clang toolchain upgrade, check if it's fixed and we can re-enable thinlto on mac
```

Comment:
I'm not entirely sure this is needed, since (AFAIK) jemalloc's build isn't going to be producing executables or shared libraries (and therefore invoking the LTO bits of the linker), but at the very least this cannot hurt things, so we'll leave it in for the time being.